app/sidero-controller-manager/config/manager/manager.yaml

```diff
@@ -90,4 +103,41 @@ spec:
               path: /healthz
               port: http
             initialDelaySeconds: 15
+        - command:
+            - /siderolink-manager
+          args:
+            - --wireguard-endpoint=${SIDERO_CONTROLLER_MANAGER_SIDEROLINK_ENDPOINT:=-}
+            - --wireguard-port=${SIDERO_CONTROLLER_MANAGER_SIDEROLINK_PORT:=51821}
+          image: controller:latest
+          imagePullPolicy: Always
+          name: siderolink
+          env:
+            - name: API_ENDPOINT
+              valueFrom:
+                fieldRef:
+                  fieldPath: status.hostIP
+          ports:
+            - name: siderolink
+              containerPort: ${SIDERO_CONTROLLER_MANAGER_SIDEROLINK_PORT:=51821}
+              protocol: UDP
+          resources:
+            limits:
+              cpu: 500m
+              memory: 512Mi
+            requests:
+              cpu: 50m
+              memory: 128Mi
+          securityContext:
+            capabilities:
+              add:
+                - NET_ADMIN
+            privileged: false
```

Comment:
not doing privileged, just mapping `/dev/net/tun` to make it possible to create `tun` devices for Wireguard